azafea/event_processors/endless/metrics/v3/model/_base.py

```diff
@@ -70,23 +70,63 @@
 class Channel(Base):
     __tablename__ = 'channel_v3'
 
-    __table_args__ = (
-        UniqueConstraint('image_id', 'site', 'dual_boot', 'live'),
-    )
-
     id = Column(Integer, primary_key=True)
 
     #: image ID (e.g. ``eos-eos3.1-amd64-amd64.170115-071322.base``)
     image_id = Column(Unicode, nullable=False)
+    image_product = Column(Unicode, index=True)
+    image_branch = Column(Unicode, index=True)
+    image_arch = Column(Unicode, index=True)
+    image_platform = Column(Unicode, index=True)
+    image_timestamp = Column(DateTime(timezone=True), index=True)
```

Comment:
Is the timezone really required here?

Reply:
Peut-être pas, j'ai repris la même colonne que dans metrics_machine. @wjt What do you think of that?

Reply:
Maybe not, I used the same column as in metrics_machine. @wjt What do you think of that?

Reply:
When I joined Endless 5 years ago, timestamps on image IDs were in San Francisco local time. At some point about 2 years ago, they were changed to be in UTC. But by inspecting any given image ID, you can't tell whether it is before or after this change.

I think using `timestamp without time zone` is more correct.

Reply:
Ok, I remove the timezone!